02-oop-inheritance/functions.js

```diff
@@ -0,0 +1,108 @@
+var Movie = function (title, duration, director) {
+	
+		this.title = title;
+		this.duration = duration;
+		this.director = director;
+		
+
+
+};
+
+Movie.prototype = {
+	
+
+
+	play:function(){ MovieObserver.playing(this.title); },
+	stop:function(){ MovieObserver.stopping(this.title); },
+	set:function(attr, value){ this[attr] = value; },
+	get:function(attr){
+		console.log(this[attr]); }
+};
+
+// Observer
+
+var MovieObserver = (function () {
```

Comment:
I don't think you need this to be an IIFE. You don't use private variable, therefore it could easily be:
var MovieObserver = {
  playing: function () {...},
  stopping: function () {...}
};